src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -124,11 +124,19 @@ public override void Convert(Stream input, IResultLogWriter output, OptionallyEm
 
             if (!string.IsNullOrWhiteSpace(_originalUriBasePath))
             {
-                var uri = new Uri(_originalUriBasePath);
-                run.OriginalUriBaseIds = new Dictionary<string, FileLocation>
+                if (_originalUriBasePath.StartsWith("/") &&
+                    _invocation.GetProperty("Platform") == "Linux")
                 {
-                    { FileLocationUriBaseId, new FileLocation { Uri = uri } }
-                };
+                    _originalUriBasePath = "file:/" + _originalUriBasePath;
```

Comment:
>/ [](start = 49, length = 1)

Reading RFC 3986, I think this should be just `file:`, not `file:/`.

Reply:
Tried that, Uri ctor throws.

---
In reply to: [253101090](https://github.com/Microsoft/sarif-sdk/pull/1254#discussion_r253101090) [](ancestors = 253101090)